TabulateSmarterTestContentPackage/Tabulator.cs

```diff
@@ -2289,6 +2328,17 @@ static string ToDependsOnString(string itemId, string dependsOnId)
             return string.Concat(itemId, "~", dependsOnId);
         }
 
+        internal bool ReportMissingImgAltTags(string input)
+        {
+            const string imgAltMatcherPattern = @"<\w*img[^>]*>";
+            var result = Regex.Match(input, imgAltMatcherPattern);
+            return result.Success
+                && !string.IsNullOrEmpty(result.Value) 
+                && (!result.Value.Contains("alt=")                         // There is an img tag, but not alt tag
+                || result.Value.Contains("alt=\"\"")                       // Empty alt tag with double quotes
+                || result.Value.Contains("alt=\'\'"));                     // Empty alt tag with single quotes
```

Comment:
@"alt(\s*\t*)=(\s*\t*)['""]\w+['""]" is what i ended up with (passed my unit tests).